packages/commonwealth/client/scripts/views/modals/update_proposal_status_modal.tsx

```diff
@@ -233,6 +236,25 @@ export const UpdateProposalStatusModal = ({
     onModalClose();
   };
 
+  const handleRemoveProposal = async () => {
+    try {
+      await deleteThreadLinks({
+        communityId: app.activeChainId(),
+        threadId: thread.id,
+        links: [
+          {
+            source: LinkSource.Snapshot,
+            identifier: initialSnapshotLinks?.[0]?.identifier ?? '',
+          },
+        ],
+      });
+      onModalClose();
+    } catch (error) {
+      console.log(error);
+      throw new Error('Failed to remove linked proposal');
```

Comment:
if you throw error, something up in the tree needs to catch it and handle it. I think in this case we should use `notifyError` isntead